app/controllers/home_controller.rb

```diff
@@ -19,15 +19,17 @@ def dashboard
       user_id: current_user.id,
       recent: true,
       include_private: true,
-      include_adult: !current_user.sfw_filter?
+      include_adult: !current_user.sfw_filter?,
```

Comment:
haha it does provide that but it also makes it unclear where the end of the list is and is a _strange_ style in that nobody uses it.  We should probably disable it.